tests/SimplifyTest.elm

```diff
@@ -5717,6 +5717,22 @@ a = left ++ ([ b ] ++ c)
 """
                     |> Review.Test.run ruleWithDefaults
                     |> Review.Test.expectNoErrors
+        , test "should replace String.fromList [ b, c ] ++ String.fromList [ d, e ] by String.fromList [ b, c , d, e ]" <|
+            \() ->
+                """module A exposing (..)
+a = String.fromList [ b, c ] ++ String.fromList [ d, e ]
+"""
+                    |> Review.Test.run ruleWithDefaults
+                    |> Review.Test.expectErrors
+                        [ Review.Test.error
+                            { message = "++ on literal strings can be turned into a single literal string"
```

Comment:
Here we kind of see why literal isn't a perfect name for this. A string literal just isn't the same as "String.fromList on a list literal".
Maybe we can add something like `fromListLiteralDescription : String` to the `FromListProperties` and use that?

Reply:
Yeah, that sounds good 👍

Reply:
I decided to go with "X.fromList call" to clear out the confusion. Would "X.fromList construction" (or something else) be better? 🤔

Reply:
That sounds so, so much better. Good job!